src/components/RichTextEditor/RichTextEditor.spec.tsx

```diff
@@ -32,6 +32,15 @@ const RichTextWithClearButton: FC<Pick<RichTextEditorProps, "value">> = ({ value
 };
 
 describe("RichTextEditor Component", () => {
+    beforeEach(() => {
+        const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/;
```

Comment:
Consider Renaming 'resizeObserverLoopErrRe' to 'resizeObserverLoopErrRegEx' or 'regExResizeObserverLoopErr'